Dockerfile

```diff
@@ -0,0 +1,29 @@
+# gammapy docker container
+FROM continuumio/miniconda3:latest
```

Comment:
Using miniconda inside docker will result in much larger docker images than say using ubuntu 20.04, installing the system python (which is 3.8).

Reply:
The installation process relies on building previously a conda environment described in the `environment-dev.yml`, so `conda` command is needed.